ci/packages/suite-web.yml

```diff
@@ -76,10 +76,16 @@ suite-web deploy dev:
   variables:
     COMPOSE_PROJECT_NAME: $CI_JOB_ID
     COMPOSE_FILE: ./docker/docker-compose.suite-ci.yml
+    # corresponds with the NextJs assetPrefix. Tells Cypress where is the index of application
     CYPRESS_ASSET_PREFIX: /web
     CYPRESS_baseUrl: ${DEV_SERVER_URL}/suite-web/${CI_BUILD_REF_NAME}
+    # should tests do snapshot testing
     CYPRESS_SNAPSHOT: 0
+    # reporter url
     TRACK_SUITE_URL: https://track-suite.herokuapp.com
+    # when debugging or developing tests it does not make sense to have retries,
+    # in other cases retries are useful to avoid occasional failures due to flaky tests
+    ALLOW_RETRY: 1
```

Comment:
Btw gitlab can also do that which might be better to see which tests are flaky? But JFYI, no need to change anything.

Reply:
this is custom retry functionality controlled by run_tests.js file. Gitlab can rerun the entire job but this is suboptimal when I need to rerun only a single test.